app/src/main/res/layout/header_today_screen.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:id="@+id/headerLinearLayout"
+    android:orientation="vertical"
+    android:background="#F0EFFF">
+    <LinearLayout
+        android:id="@+id/home_screen_first_part"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        android:layout_marginBottom="54dp"
+        >
+
+    <TextView
+        android:id="@+id/palermo_sic"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="24dp"
+        android:layout_marginTop="42dp"
+        android:layout_marginEnd="24dp"
+        android:text="@string/palermo_sic"
+        android:textAppearance="@style/palermo_sic"
+        />
+    <LinearLayout
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        >
+        <TextView
+            android:layout_width="184dp"
+            android:layout_height="25dp"
+            android:layout_marginStart="24dp"
+            android:layout_marginTop="2dp"
+            android:text="Oggi,"
```

Comment:
text here in Italian? I think that we need  to use strings, and set text programmatically.